test/desugar/ref-func.txt

```diff
@@ -0,0 +1,14 @@
+;;; TOOL: wat-desugar
```

Comment:
Thanks!

I think maybe this should live in test/roundtrip/reference-types.txt?

lgtm with that change.

Reply:
Updated :+1: